packages/model-fragments/lib/fragments/model.js

```diff
@@ -241,5 +135,29 @@ function getActualFragmentType(declaredType, options, data) {
   return actualType || declaredType;
 }
 
+// Returns the internal model for the given record/fragment
+export function internalModelFor(record) {
+  var internalModel = record._internalModel;
+
+  // Ensure the internal model has a fragments hash, since we can't override the
+  // constructor function anymore
+  if (!internalModel._fragments) {
+    internalModel._fragments = create(null);
+  }
+
+  return internalModel;
+}
```

Comment:
@devinus makes sense, I had just assumed that the ED object polyfills were full implementations.